### DISCUSSION TITLE
Duplicate, outdated and unused theme-base directory in Prime NG Ultimate Theme 17 (also in 16)

### DISCUSSION THREAD

**Discussion by SiebeDom** [primefaces/community#1187] [PrimeNG Templates] [answered]
Hello,

I noticed that: src/assets/layout/styles/theme/theme-base/theme-base is duplicate, outdated and unused.
It can cause confusion.

How do I come to this conclusion:

- Duplicate: there are 2 theme-base folders: src/assets/layout/styles/theme/theme-base (correct one) and src/assets/layout/styles/theme/theme-base/theme-base (incorrect one)
- Unused: 
  - When removed src/assets/layout/styles/theme/theme-base (and recompile scss) and then run the Ultima theme is still looking the same as before
  - When I look for references of src/assets/layout/styles/theme/theme-base/theme-base/_components.scss in Intellij, no references can be found
- Outdated: 
  - src/assets/layout/styles/theme/theme-base/theme-base/_components.scss does not have CSS layers
  - When I compare theme-base of https://github.com/primefaces/primeng-sass-theme with I see that only minimal differences with src/assets/layout/styles/theme/theme-base (the correct one)

If my observation is correct can directory: src/assets/layout/styles/theme/theme-base/theme-base/ be removed in one of the next releases of the Prime NG Ultima theme?

**Comment by cetincakiroglu** [comment, marked as answer]
Hi,

We couldn't find the duplicate folder. Please see the screenshot from the latest version downloaded from the store.

![Screenshot 2024-02-19 at 10 24 22](https://github.com/primefaces/community/assets/69278826/a25d756c-33a0-4ccd-9557-8794b518551d)

**Reply by SiebeDom** [reply]
Hello,

I see now.
Ultima 16 had this duplicate problem.
It was fixed in Ultima 17.
We copied over the files from 17.
If we upgrade in the future it's best to delete everything in src/assets/layout (except for our customizations) and then copy it over from the latest version.

Thanks for your answer!

**Reply by AshishFreak** [reply]
Hi @SiebeDom  can you please share ultima theme with me ?
